index.html

```diff
@@ -40,6 +41,160 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
         <li><a href="#" title="Angellist"><img src="img/angellist.svg" alt="angellist logo"></a></li>
       </ul>
     </section>
+     <!-- Work -->
+    <section id="work">
+      <div class="work-first flex-column">
+        <div class="work-description flex-column">
+          <div> <img src="img/Portfolio-1.png" alt=""></div>
+          <div>
+            <div class="specification">
+              <h2>Hossain</h2>
+              <div class="flex">
+                <h4>CANOPY</h4>
+                <!-- <p>&#x2022;</p>
+                <P> Full Stack Dev</P>
+                <p>&#x2022;</p>
+                <p>2023</p> -->
+                <ul class="flex">
+                  <li>Full Stack Dev</li>
+                  <li>2023</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description flex">
+          <button class="language-btn"> <p>html</p></button>
+          <button class="language-btn"><p>css</p></button>
+          <button class="language-btn"><p>Javascript</p></button>
+        </div>
+        <button class="see-project-btn">See Project</button>
```

Comment:
- [ ] You did great using the button tag here. :+1: But I think there are still some styling issues you need to update here. 
- [ ] Currently, the background color of your button is not the same as the one in Figma, I'll suggest you get the correct color and update this as replicating the given design is one of the requirements for this milestone. :+1: 
- [ ] Also your buttons do not have the interactive properties implemented, You are meant to implement the `hover and pressed` state of your button with the correct background color and text color. I'll suggest you update this too. :+1: :100: 

- - HOVER AND PRESSED STATE FROM DESIGN:
![Screenshot from 2023-06-07 09-16-03](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/107632780/15649450-132c-4ef2-af7e-bba644996e85)